db/config.json

```diff
@@ -20,5 +20,18 @@
 			"min": 0,
 			"idle": 10000
     }
-  }
+  },
+	"production-migrate": {
+		"dialect": "postgres",
+		"use_env_variable": "PG_DATABASE_URL",
```

Comment:
So the difference here is in the database url, right? DATABASE_URL goes through the PG Bouncer and PG_DATABASE_URL goes directly to postgres, correct?

Reply:
Correct.

Reply:
This is just a workaround for now until pgbouncer permissions are fixed with the sidecar.